packages/share/example/lib/main.dart

```diff
@@ -69,11 +91,30 @@ class DemoAppState extends State<DemoApp> {
                               // a RenderObjectWidget. The RaisedButton's RenderObject
                               // has its position and size after it's built.
                               final RenderBox box = context.findRenderObject();
-                              Share.share(text,
-                                  subject: subject,
-                                  sharePositionOrigin:
-                                      box.localToGlobal(Offset.zero) &
-                                          box.size);
+
+                              if (imagePaths.isNotEmpty) {
+                                if (imagePaths.length == 1) {
+                                  await Share.shareFile('',
```

Comment:
Shouldn't this share `imagePaths.first` instead of an empty string?

Reply:
Not sure why I never noticed this. But removing this anyway in favour of `shareFiles()`